media-sound/helm/files/helm-0.9.0-fix-atomic.patch

```diff
@@ -0,0 +1,66 @@
+diff --git a/builds/linux/LV2/Makefile.binary b/builds/linux/LV2/Makefile.binary
+index 6559f5f..4a82cdd 100644
+--- a/builds/linux/LV2/Makefile.binary
++++ b/builds/linux/LV2/Makefile.binary
+@@ -45,7 +45,7 @@ ifeq ($(CONFIG),Debug)
+ 
+   JUCE_CFLAGS += $(JUCE_CPPFLAGS) $(TARGET_ARCH) -fPIC -g -ggdb -O0 $(DEBCXXFLAGS) -ffast-math $(SIMDFLAGS) $(CFLAGS)
+   JUCE_CXXFLAGS += $(CXXFLAGS) $(JUCE_CFLAGS) -std=c++11 $(CXXFLAGS)
+-  JUCE_LDFLAGS += $(TARGET_ARCH) -L$(JUCE_BINDIR) -L$(JUCE_LIBDIR) -Wl,--no-undefined -L/usr/X11R6/lib/ $(shell pkg-config --libs alsa freetype2 libcurl x11 xext xinerama) -lGL -ldl -lpthread -lrt $(DEBLDFLAGS) -ffast-math $(SIMDFLAGS) $(LDFLAGS)
++  JUCE_LDFLAGS += $(TARGET_ARCH) -L$(JUCE_BINDIR) -L$(JUCE_LIBDIR) -Wl,--no-undefined -L/usr/X11R6/lib/ $(shell pkg-config --libs alsa freetype2 libcurl x11 xext xinerama) -lGL -ldl -lpthread -latomic -lrt $(DEBLDFLAGS) -ffast-math $(SIMDFLAGS) $(LDFLAGS)
```

Comment:
Unfortunately, unconditionally adding -latomic isn't right either for say, pure LLVM builds. We're better off doing `append-atomic-flags` in the ebuild in lieu of a proper configure test.

Reply:
`append-atomic-flags` seems doesn't works for this Makefile